docs/source/using_the_ve/configuration/config.md

```diff
@@ -198,15 +198,35 @@ All models (except `core`) accept a boolean configuration option, `static`, that
 indicates if such a model should be updated every iteration (`static=false`, the
 default behaviour) or not.
 
-If `static=true`, there are several possibilities depending on the data
-variables available in the [`Data` object](../data/data.md):
-
-- For the initialisation, if the variables that are populated are:
-  - **All present**: The model initialisation process is bypassed.
-  - **None present**: The model initialisation process runs normally.
-- For the update, if the variables that are populated are:
-  - **All present**: Then the update process is bypassed.
-  - **None present**: The update *is run just once*, keeping the same values for those
-  variables throughout the simulation.
+If `static=true`, the initial model behaviour depends on the data variables available in
```

Comment:
Maybe 'provided at the start of the simulation' instead of 'available? Just to make super clear what the workflow is?